torch/csrc/jit/serialization/import_source.cpp

```diff
@@ -578,7 +578,9 @@ void SourceImporterImpl::importClass(
       /*propResolvers=*/{},
       methods,
       method_resolvers,
-      &self);
+      &self,
+      false,
```

Comment:
Consider specifying which argument `false` is passed to like what line 578 is doing.

Reply:
done